UploadServiceLogo/handler.ts

```diff
@@ -108,65 +102,58 @@ export function UpdateServiceLogoHandler(
 
     const bufferImage = Buffer.from(logoPayload.logo, "base64");
     const lowerCaseServiceId = serviceId.toLowerCase();
-    return fromEither(
-      tryCatch(() => UPNG.decode(bufferImage)).foldL(
+    return pipe(
+      O.tryCatch(() => UPNG.decode(bufferImage)),
+      O.fold(
         () =>
-          left<IResponseErrorValidation, UPNG.Image>(
+          E.left<IResponseErrorValidation, UPNG.Image>(
             imageValidationErrorResponse()
           ),
-        img => right<IResponseErrorValidation, UPNG.Image>(img)
-      )
-    )
-      .chain(image =>
-        fromPredicateT(
+        img => E.right<IResponseErrorValidation, UPNG.Image>(img)
+      ),
+      TE.fromEither,
+      TE.chain(image =>
+        TE.fromPredicate(
           (img: UPNG.Image) => img.width > 0 && img.height > 0,
           () => imageValidationErrorResponse()
         )(image)
-      )
-      .foldTaskEither<
-        IResponseErrorValidation | IResponseErrorInternal,
-        // eslint-disable-next-line @typescript-eslint/ban-types
-        IResponseSuccessRedirectToResource<{}, {}>
-      >(
-        imageValidationError => fromLeft(imageValidationError),
-        () =>
+      ),
```

Comment:
```typescript
      TE.chain(
        TE.fromPredicate(
          (img: UPNG.Image) => img.width > 0 && img.height > 0,
          () => imageValidationErrorResponse()
        )
      )
```

Reply:
fixed in https://github.com/pagopa/io-functions-admin/pull/165/commits/409be2e4d1eaf1271b27f010aa2cacd12e7ce248